Makefile

```diff
@@ -54,7 +57,11 @@ test: clean-cov fmt vet $(GINKGO)
 ## install: Build and install kuadrantctl binary ($GOBIN or GOPATH/bin)
 .PHONY : install
 install: fmt vet
+ifneq ($(GIT_HASH),$(GIT_DIRTY))
```

Comment:
why comparing those?

the pseudo-code should be as follows:
```
git_sha = git rev-parse --short=7 HEAD
if no error:
    dirty =  git diff --stat
    if dirty:
        git_hash = ${git_sha}-dirty
    else:
        git_hash = ${git_sha}
    end
else
    git_hash = env(GITHUB_SHA) or "NO_SHA"
end
```

Reply:
Sure, I believe there might be a suggestion below which hints at why there is need for `GIT_DIRTY` to be a boolean, and not a string. I'll take what you've said as context for the below suggestion.